README.md

```diff
@@ -139,3 +139,11 @@ fzf-history() {
 zle -N fzf-history
 bindkey '^R' fzf-history
 ```
+
+### Open random manpage
+
+Credit: [Mcotocel](https://www.github.com/Mcotocel)
+
+```
+man -k . | awk '{print $1}' | shuf -n 1 | xargs kitty man
```

Comment:
Would you be able to make it not open a new term?  just `xargs man` should be fine.

Reply:
actually nvm, i can do that.